src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -208,7 +206,7 @@ describe('non-empty successful affected clusters table', () => {
     it('checkbox is unselected when a row is unselected', () => {
       cy.ouiaId(BULK_SELECT).find('input').click().should('be.checked');
       // removing one row unselects it
-      cy.get(ROOT)
+      cy.get('table')
```

Comment:
Isn't it better to keep it under one variable ROOT?

Reply:
My argument was rather about using a constant instead of a value directly so that you don't need to change each occurrence once you decide to replace the `'table'` with another selector. But since it's least likely, I think it's not an issue :)

Reply:
It is a fair point. I will replace them with a new global